ChatSecure/Classes/View Controllers/OTRMessagesGroupViewController.m

```diff
@@ -44,6 +44,11 @@ - (void)setupGroupChat:(NSArray <NSString *>*)buddies account:(OTRAccount *)acco
     [self setThreadKey:self.threadKey collection:[OTRXMPPRoom collection]];
 }
 
+- (nullable OTRAccount *)accountWithTransaction:(nonnull YapDatabaseReadTransaction *)transaction {
```

Comment:
@N-Pex I think this already exists in the superclass and you could just expose the method in a private header / class extension?